docs/mmteb/points.md

```diff
@@ -27,6 +27,7 @@
 | cassanof          |              | 6           |          |                     | 1          | 1              |           |                |          |               |
 | taidnguyen        |              | 14          |          |                     |            |                |           |                |          |               |
 | orionw         |              |            |          |                     |      2      |              |           |                |          |               |
+| Rysias |              |     24      |          |                    |          |                |         |                |          |              |
```

Comment:
The points would be:

- 2 for the dataset (reformulated as multilingual)
- 4x3 for new language x task combination (Danish already has a clustering task)

We could probably extend this dataset to more languages, but let us just test it out in this stage first.

You should also add a point for the review (to me / KennethEnevoldsen).

Reply:
roger!